wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/button/Trigger.java

```diff
@@ -205,7 +205,7 @@ public void run() {
   }
 
   /**
-   * Toggles a command when the condition changes from `true` to the low state.
+   * Toggles a command when the condition changes from `true` to 'false'.
```

Comment:
These need to be backticks, not quotes.

Reply:
Ah, yeah, you're right. I'll fix that now